queries.sql

```diff
@@ -7,3 +7,26 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE NOT name = "Gabumon";
 SELECT * FROM animals WHERE  10.4 <=weight_kg AND  weight_kg <=17.3 ;
 
+-- How many animals are there?
+SELECT COUNT(*)
+FROM animals;
+-- How many animals have never tried to escape?
+SELECT COUNT(escape_attempts)
+FROM animals
+WHERE escape_attempts =0;
+-- What is the average weight of animals?
+SELECT AVG(weight_kg)
+FROM animals;
+-- Who escapes the most, neutered or not neutered animals?
+SELECT MAX(escape_attempts)
+FROM animals
+WHERE neutered = TRUE OR neutered = FALSE;
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species, MAX(weight_kg),MIN(weight_kg)
+FROM animals
+GROUP BY species;
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species,AVG(escape_attempts)
+FROM animals
+WHERE date_of_birth>='1990-01-01' AND date_of_birth<='2001-01-01' 
```

Comment:
- [ ] Ending your check on `2001-01-01` doesn't satisfy the check for those animals born between 1990 and 2000. You should set your ending check to the last day and the last month of the year 2000.

See example: 
```
SELECT species, AVG(escape_attempts) FROM animals
GROUP BY species
WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/31/12'
```

Reply:
I don't see the point? it will be less than the first day and first month of 2000 what is the problem with that?

Reply:
The point here is an animal born on this exact date '2001-01-01'  will be counted in the query.
We don't need to get animals born on the first day and first month of 2001. The check should end on December 31, 2000.
I hope this is clear?